### PR TITLE
ZKUI-258 Remove deprecated colors and icons

### PR DESCRIPTION
**Pull request by ghivert**

### CODE REVIEW

**Comment by ghivert** [comment]
Boop

File: src/react/ui-elements/Help.tsx [hunk 1]
```diff
@@ -1,21 +1,37 @@
 import { Tooltip } from '@scality/core-ui';
-import React from 'react';
-import type { Node } from 'react';
+import * as Core from '@scality/core-ui';
+import { IconName } from '@scality/core-ui/dist/components/icon/Icon.component';
```
Comment (ChengYanJin) [inline src/react/ui-elements/Help.tsx:2]: With the latest release of core-ui `v0.47.0`, we have the IconHelp integrated.  
So let's remove this Help component in zenko-ui and use the one in core-ui ;)

File: src/react/ui-elements/CopyButton.tsx [hunk 2]
```diff
@@ -1,12 +1,12 @@
+import { Icon } from '@scality/core-ui';
```
Comment (ChengYanJin) [inline src/react/ui-elements/CopyButton.tsx]: Just FYI, We have `CopyButton` integrated  within core-ui as well.

**Review by ChengYanJin — COMMENTED** [review]
We could use the IconHelp and CopyButton directly from core-ui with the latest release. 
Other than this LGTM !

**Comment by ghivert** [comment]
Boop

**Review by ChengYanJin — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**